### PR TITLE
Implemented the changes to be able to send Html / Multipart Emails

### PR DESCRIPTION
**Pull request by flyrmi**
This PR adds a `isHtml: Boolean` to the `send` & `sendAsync` methods of `fm.common.EmailSender`.

Notes:
  * I choose this over duplicating code and creating `sendHtml` & `sendHtmlAsync`, but I have no preference in case you prefer to keep the other methods as-is.
  * As for the `if (isHtml) { ... }`: I think it's also possible to just set a header so that it is being displayed as Html which might be cleaner to read. Any thoughts?

### CODE REVIEW

File: jvm/src/main/scala/fm/common/EmailSender.scala [hunk 1]
```diff
@@ -100,4 +224,27 @@ final case class EmailSender(user: String, pass: String, host: String) {
       transport.close()
     }
   }
+
+  private def createMultipart(plaintextBody: String, htmlBody: String): MimeMultipart = {
+    val multipart: MimeMultipart = new MimeMultipart()
+
+    // According to the RFC for Multipart content, the plainest format should be set first and the richest format last
```
Comment (tpunder) [inline jvm/src/main/scala/fm/common/EmailSender.scala]: Typo: `plainest ` should be `plaintext `

Comment (flyrmi) [reply jvm/src/main/scala/fm/common/EmailSender.scala]: Oh I just copied that part from w3. They ranked it from plainiest to richest.

**Comment by er1c** [comment]
@flyrmi did you eventually this this to make sure they render properly in gmail/outlook/etc?

**Comment by flyrmi** [comment]
> @flyrmi did you eventually this this to make sure they render properly in gmail/outlook/etc?

Not for GMail, but for Outlook (Desktop and Web). Since we're just passing the HTML and don't validate it or so, it needs to be in proper formatting anyway to be displayed correct.

**Pull request merged**